unix-process/creator.c

```diff
@@ -0,0 +1,26 @@
+#include "employee.h"
+#include <stdio.h>
+#include <stdlib.h>
+
+int main(int argc, char* argv[]) {
```

Comment:
Split input and output in different modules and remove all unnecessary empty lines